pom.xml

```diff
@@ -152,8 +165,531 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
-      <!-- pure test deps -->
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>proto-google-cloud-bigtable-v2</artifactId>
+        <version>${generated-proto-beta.version}</version>
```

Comment:
Actually I think it might make sense to have the proto/grpc generated classes in the BOM - let's follow up with the relevant parties to confirm.